test/corpus/actions.txt

```diff
@@ -287,6 +301,20 @@ range else
         (text)
         (text)))
 
+=============================================
+Invalid range
+=============================================
+{{range $var=5}} t1 {{ end }}
```

Comment:
Very interesting,

this is not allowed:

```
{{ $var:=list }}
{{range $var=list}} t1 {{ end }}
```

But this is (even executes):
```
{{range $var:=list}} t1 {{ end }}
```

Reply:
Yeah, the second is an element range. I added support for that in #32